RevenueCatUI/Templates/V2/ViewModelHelpers/PackageValidator.swift

```diff
@@ -19,12 +19,12 @@ import RevenueCat
 @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
 class PackageValidator {
 
-    typealias PackageInfo = (package: Package, isSelectedByDefault: Bool)
+    typealias PackageInfo = (package: Package, isSelectedByDefault: Bool, promotionalOfferProductCode: String?)
```

Comment:
This is out of scope for this PR, but I've been thinking if we should support a list of promo offers codes, instead of a single one... So basically the first code would have the highest priority if it's eligibile, if not fall back to the next... Just thinking about making the system more flexible, but I think this is good for now :)